core/link.c

```diff
@@ -2269,20 +2269,21 @@ shift_links_to_new_fragment(dcontext_t *dcontext,
         /* Change the entrance stub to point to the trace head routine again
          * (we only shift to coarse trace heads).
          */
-        coarse_info_t *info = get_fragment_coarse_info(new_f);
```

Comment:
how about `new_f_info`? I think that is more descriptive

Reply:
good proposal, changed